### PR TITLE
Add column type to typescript types

### PR DESCRIPTION
**Pull request by Samlinxia**
Add the column type to Select statement

### CODE REVIEW

**Comment by Samlinxia** [comment]
@taozhi8833998 : please take a look when you are available.

File: index.d.ts [hunk 1]
```diff
@@ -49,12 +49,21 @@ export interface InsertReplaceValue {
   type: 'expr_list';
   value: any[];
 }
+export interface AggrFunc {
+  type: 'aggr_func';
+  name: string;
+  args: ColumnRef | AggrFunc | null;
```
Comment (taozhi8833998) [inline index.d.ts:55]: it would be greater if you could add `count(*)` parsed result

**Comment by taozhi8833998** [comment]
@Samlinxia Thank you for your pr, I am reviewing your code, hope you could make a little change to your commit to cover more type. Thanks again for ur pr.

**Comment by taozhi8833998** [comment]
@Samlinxia Thank you for ur pr!! I have merged it, and I will publish the feature today.

**Comment by taozhi8833998** [comment]
@Samlinxia `2.3.0` version has been published, and it includes the codes you commit, try it. Thanks for ur pr.

**Pull request merged**